app/views/notification_mailer/notification_email.html.slim

```diff
@@ -0,0 +1,13 @@
+h4
+  = I18n.t('notification_mailer.notification.greeting') + ', ' +  @user.name
```

Comment:
You should be using translation variables too embed the name within the translatable string.